src/live_restore/live_restore_private.h

```diff
@@ -44,55 +44,55 @@ struct __wt_live_restore_file_handle {
         bool complete;
 
         /* We need to get back to the file system when checking for tombstone files. */
-        WT_LIVE_RESTORE_FS *back_pointer;
+        WTI_LIVE_RESTORE_FS *back_pointer;
 
         /*
          * The hole list tracks which ranges in the destination file are holes. As the migration
          * continues the holes will be gradually filled by either data from the source or new
          * writes. Holes in these extents should only shrink and never grow.
          */
-        WT_LIVE_RESTORE_HOLE_NODE *hole_list_head;
+        WTI_LIVE_RESTORE_HOLE_NODE *hole_list_head;
     } destination;
 
     WT_FS_OPEN_FILE_TYPE file_type;
     WT_RWLOCK ext_lock; /* File extent list lock */
 };
 
 /*
- * WT_WITH_LIVE_RESTORE_EXTENT_LIST_WRITE_LOCK --
+ * WTI_WITH_LIVE_RESTORE_EXTENT_LIST_WRITE_LOCK --
  *     Acquire the extent list write lock and perform an operation.
  */
-#define WT_WITH_LIVE_RESTORE_EXTENT_LIST_WRITE_LOCK(session, lr_fh, op) \
-    do {                                                                \
-        __wt_writelock((session), &(lr_fh)->ext_lock);                  \
-        op;                                                             \
-        __wt_writeunlock((session), &(lr_fh)->ext_lock);                \
+#define WTI_WITH_LIVE_RESTORE_EXTENT_LIST_WRITE_LOCK(session, lr_fh, op) \
+    do {                                                                 \
+        __wt_writelock((session), &(lr_fh)->ext_lock);                   \
+        op;                                                              \
+        __wt_writeunlock((session), &(lr_fh)->ext_lock);                 \
     } while (0)
 
 typedef enum {
     WT_LIVE_RESTORE_FS_LAYER_DESTINATION,
```

Comment:
should these be WTI?

Reply:
Fixed in 883a8834fe7cdfe53028a33baad62d17639b9ae3